src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```
@@ -80,25 +50,25 @@ $schedule-btn--height: 51px;
     background-color: opaque-black(1);
   }
 
+  &__weekdayButton {
```

We camel case only the B(lock) in this project's BEM to be same as the components. All else is kebob-case

```suggestion
  &__weekday-button {
```

---

fixed in 2c16847b0928ba957a737453f7b445db887dda23